labgrid/driver/powerdriver.py

```diff
@@ -271,27 +271,37 @@ def get(self):
 @attr.s(eq=False)
 class YKUSHPowerDriver(Driver, PowerResetMixin, PowerProtocol):
     """YKUSHPowerDriver - Driver using a YEPKIT YKUSH switchable USB hub
-        to control a target's power - https://www.yepkit.com/products/ykush"""
+        to control a target's power - https://www.yepkit.com/products/ykush.
+        Uses ykushcmd tool to control the hub."""
     bindings = {"port": "YKUSHPowerPort", }
```

Comment:
This needs to explicitly bind to `NetworkYKUSHPowerPort` as well.
```suggestion
    bindings = {"port": {"YKUSHPowerPort", "NetworkYKUSHPowerPort"} }
```
See for example the `FlashromDriver`.